samples/release-management/csharp/ReleaseManagement/Bots/ActivityBot.cs

```diff
@@ -0,0 +1,44 @@
+﻿// <copyright file="ActivityBot.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+namespace ReleaseManagement.Bots
+{
+    using Microsoft.Bot.Builder;
+    using Microsoft.Bot.Schema;
+    using ReleaseManagement.Models;
+    using ReleaseManagement.Services;
+    using System.Collections.Concurrent;
+    using System.IO;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public class ActivityBot : ActivityHandler
+    {
+        private readonly ConcurrentDictionary<string, ReleaseManagementTask> taskDetails;
+        private readonly ICardFactory cardFactory;
+
+        public ActivityBot (ConcurrentDictionary<string, ReleaseManagementTask> taskDetails, ICardFactory cardFactory)
+        {
+            this.taskDetails = taskDetails;
+            this.cardFactory = cardFactory;
+        }
+        protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivity> turnContext, CancellationToken cancellationToken)
+        {
+            var replyText = $"Echo: {turnContext.Activity.Text}";
+            await turnContext.SendActivityAsync(MessageFactory.Text(replyText, replyText), cancellationToken);
```

Comment:
Do we need this.

Reply:
As such not required, just kept it in case.

Reply:
Removed.